walletconsole/lib/Address.cpp

```diff
@@ -0,0 +1,112 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#include "Address.h"
+
+#include "WalletConsole.h"
+#include "Coin.h"
+#include "Data.h"
+#include "PrivateKey.h"
+#include "HexCoding.h"
+#include "HDWallet.h"
+
+#include <iostream>
+#include <vector>
+#include <cassert>
+
+namespace TW::WalletConsole {
+
+using namespace std;
+
+bool Address::addrpub(const string& coinid, const string& pubkey_in, string& res) {
```

Comment:
let's always use CamelCase? addrPub, addrPriv etc

Reply:
You mean in command names as well?  OK, they are more readable that way.  Comparison is case-insensitive anyways :)